web/client/components/data/featuregrid/filterRenderers/AttributeFilter.jsx

```diff
@@ -50,7 +50,7 @@ class AttributeFilter extends React.PureComponent {
             booleanOperators: ["="],
             defaultOperators: ["=", ">", "<", ">=", "<=", "<>", "isNull"],
             timeDateOperators: ["=", ">", "<", ">=", "<=", "<>", "><", "isNull"],
-            operator: this.props.isWithinAttrTbl ? "=" : "",
+            operator: this.props.isWithinAttrTbl ? (this.props.operator || '=') : "",
```

Comment:
@mahmoudadel54 The `this.props.operator` was introduced in a previous PR https://github.com/geosolutions-it/MapStore2/pull/9743 and in this new PR is used to apply a default value. Few questions:
- What was the role of the prop `operator` before this change? It was used?
- If the prop `operator` was existing before why do we are using it to apply a default value?

Reply:
@allyoucanmap 
this.props.operator  ---> it is used in BaseDateTimeFilter component - which inherits from AttributeFilter class component - in onChange handlers but after double checking I think I can remove it and just use the operator state

for the second point, I think again I can just use initiate the state operator based on this.props.type to implement ilike in case of string field

If Ok, I will push the code for that, thanks

Reply:
@mahmoudadel54 I would suggest to introduce a specific prop for the default value such as defaultOperator, we cannot use type because the current request is a change just to the attribute table while the usage of type could effect also the query builder.
About the removal of `this.props.operetor` we should do it only if we are 100% sure it's not really used